src/Backend/Modules/Pages/Js/Pages.js

```diff
@@ -732,7 +732,7 @@ jsBackend.pages.extras =
 
 			html = '<div id="user-template-textarea-' + key + '" class="options">';
 			html += '<p class="p0"><label>' + label + '</label></p>';
-			html += '<textarea data-ft-label="' + label + '" cols="90" rows="15">' + text + '</textarea>';
+			html += '<textarea data-ft-label="' + label + '" cols="83" rows="15">' + text + '</textarea>';
```

Comment:
Klopt, maar het is inderdaad beter om het bij ckeditor ook hetzelfde in te stellen. Gefixt in https://github.com/sumocoders/forkcms/pull/61/commits/acff922d9028baebdf12b34241b935a97046e765